lib/apimatic-core/api_call.rb

```diff
@@ -86,7 +86,6 @@ def execute
         _deserialized_response = @response_handler.endpoint_logger(@endpoint_logger)
                                                   .endpoint_name_for_logging(@endpoint_name_for_logging)
                                                   .handle(_http_response, @global_configuration.get_global_errors,
-                                                          @global_configuration.get_sdk_module,
                                                           @global_configuration.should_symbolize_hash)
```

Comment:
Blocker: The `sdk_module` property should also be removed from `GlobalConfiguration` class. As this is not needed anymore.

Reply:
done